datalake/src/main/java/com/sequenceiq/datalake/flow/dr/restore/DatalakeRestoreActions.java

```diff
@@ -65,6 +69,46 @@
     @Inject
     private SdxService sdxService;
 
+    @Bean(name = "DATALAKE_TRIGGERING_RESTORE_STATE")
+    public Action<?, ?> triggerDatalakeRestore() {
+        return new AbstractSdxAction<>(DatalakeTriggerRestoreEvent.class) {
+            @Override
+            protected SdxContext createFlowContext(FlowParameters flowParameters, StateContext<FlowState, FlowEvent> stateContext,
+                    DatalakeTriggerRestoreEvent payload) {
+                return SdxContext.from(flowParameters, payload);
+            }
+
+            @Override
+            protected void prepareExecution(DatalakeTriggerRestoreEvent payload, Map<Object, Object> variables) {
+                variables.put(OPERATION_ID, payload.getDrStatus().getOperationId());
```

Comment:
why is this needed? There is another OPERATION_ID put later.

Reply:
Yes. You are right. Thanks. Corrected!